### PR TITLE
Update UG

### LINKED ISSUE #212: UG - Update Command Summary

### LINKED ISSUE #213: UG - FAQ

### LINKED ISSUE #236: UG - Make UG more reader friendly

### PR DESCRIPTION
**Pull request by goatygoatygoat**
Update UG with FAQ, added saving and editing data features, modified command summary

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -47,7 +47,10 @@ An image of our UI is shown below!
         - [Delete](#deleting-an-entry)
```
Comment (tsh22) [inline docs/UserGuide.md]: Would it be nicer if we put 'Deleting an entry: `edelete`' kind of format for our content list? From what I saw from others' user guides haha, but up to you to change

File: docs/UserGuide.md [hunk 2]
```diff
@@ -100,8 +103,9 @@ Adds a contact into Teaching Assistant.
 
 Format: `cadd n/NAME p/NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
 
-* A contact can have any number of tags (including 0).
-* A contact can be tagged to an entry to easily filter for contacts related an entry.
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A contact can be tagged to an entry to easily filter for contacts related an entry.
```
Comment (tsh22) [inline docs/UserGuide.md]: Think this phrasing might be a bit misleading, should we remove it?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -338,22 +363,23 @@ Action | Format
...
-**Add** | `add n/NAME p/NUMBER e/EMAIL a/ADDRESS [t/TAG]`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`
-**Filter tags** | `filter KEYWORD [MORE_KEYWORDS]`
-**Edit** | `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]`
-**List** | `list`
-**Delete** | `delete INDEX`
+**Add** | `cadd n/NAME p/NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+**Find** | `cfind KEYWORD [MORE_KEYWORDS]...`
+**Filter tags** | `cfilter KEYWORD [MORE_KEYWORDS]...`
+**Edit** | `cedit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
```
Comment (tsh22) [inline docs/UserGuide.md]: Remove address field

File: docs/UserGuide.md (hunk 2 above)
Comment (goatygoatygoat) [reply docs/UserGuide.md]: If i say "an entry's name can be tagged to a contact to easily filter for contacts related an entry" is it clearer? cuz this in response to a pe bug where ppl say they cannot link contacts and entries

**Review by nicholastanvis — APPROVED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (tsh22) [reply docs/UserGuide.md]: Sure, looks good thanks!

**Review by tsh22 — APPROVED** [review]

File: docs/UserGuide.md [hunk 4]
```diff
@@ -256,29 +349,45 @@ Example(s):
...
 * `elist day`
 * `elist week`
 
-### Checking if time interval is free
+Scenario: You want to see what is in store for you today so that you can mentally prepare yourself for the busy day ahead.
+
+### Checking if time interval is free: `free`
 
 Indicates if an interval is free. If free, a message indicating that will be shown. If not, entries occupying that
 interval will be shown in the entries list.
```
Comment (tsh22) [inline docs/UserGuide.md]: entry list since we standardising haha

File: docs/UserGuide.md [hunk 5]
```diff
@@ -288,12 +397,19 @@ Example(s):
 * `free sd/ 2021-12-20 12:00 ed/ 2021-12-20 13:00` if the time interval is free, entries list will be empty and _"You're
```
Comment (tsh22) [inline docs/UserGuide.md]: entry list

File: docs/UserGuide.md [hunk 6]
```diff
@@ -288,12 +397,19 @@ Example(s):
...
 
-Format: `edelete INDEX`
+**Format**: `edelete INDEX`
+
+**Breakdown**:
+* Command word - `edelete`
+* Parameters - `INDEX`
+
 
 * `INDEX` refers to the index number shown in the displayed person list.
```
Comment (tsh22) [inline docs/UserGuide.md]: contact list

File: docs/UserGuide.md [hunk 7]
```diff
@@ -302,25 +418,66 @@ Example(s):
...
 Clears all entries that have dates before today's date.
 
-Format: `eclear`
+**Format**: `eclear`
+
+**Breakdown**:
+* Command word - `eclear`
+
+Scenario: You still have entries from the past that you no longer need, and want to remove those outdated entries
+to not clutter the entries list.
```
Comment (tsh22) [inline docs/UserGuide.md]: entry list

File: docs/UserGuide.md [hunk 8]
```diff
@@ -163,18 +209,32 @@ Example(s):
...
 
-Format: `cdelete INDEX`
+**Format**: `cdelete INDEX`
+
+**Breakdown**:
+* Command word - `cdelete`
+* Parameters - `INDEX`
+
 
 * `INDEX` refers to the index number shown in the displayed person list.
```
Comment (tsh22) [inline docs/UserGuide.md]: contact list

**Review by tsh22 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 9]
```diff
@@ -144,11 +179,22 @@ Example(s):
...
+**Format**: `cedit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [t/TAG]...`
+
+**Breakdown**:
+* Command word - `cedit`
+* Prefixes - `n/`, `p/`, `e/`, `t/`
+* Parameters - `INDEX`, `NAME`, `NUMBER`, `EMAIL`, `TAG`
+
 
 * `INDEX` refers to the index numer shown in the displayed entry list.
 * `INDEX` must be a positive integer 1,2,3, ...
```
Comment (tsh22) [inline docs/UserGuide.md:200]: probably should markup the /t

File: docs/UserGuide.md [hunk 10]
```diff
@@ -142,87 +177,136 @@ Format: `cfilter KEYWORD [MORE_KEYWORDS]...`
...
+
+**Breakdown**:
+* Command word - `eadd`
+* Prefixes - `n/`, `sd/`, `ed/`, `t/`
+* Parameters - `NAME`, `START_DATE`, `END_DATE`, `TAG`
+
 
 * `START DATE` and `END DATE` are in the format `yyyy-mm-dd HH:MM`.
 * `START DATE` should be before `END DATE`.
 * Entries cannot overlap. i.e. Entries with overlapping timings will not be added.
```
Comment (tsh22) [inline docs/UserGuide.md:267]: May need to add an extra thing saying overdue entries can still be added but the entry card will be shown as red, I'll define what an entry card means at my part

**Review by tsh22 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 10 above)
Comment (tsh22) [reply docs/UserGuide.md:267]: Sry for the scattered reviews ^^" I'm noticing stuff as i'm doing my part

Comment (goatygoatygoat) [reply docs/UserGuide.md:267]: hahah no problems :thumbsup:

**Review by lrj689 — APPROVED** [review]

**Review by tsh22 — APPROVED** [review]
LGTM

**Pull request merged**